### PR TITLE
fix url for xpdan

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: recipes-tag/xpdan/meta.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ requirements:
     - pyyaml
     - scikit-beam
     - pyfai
+    - pyxdameraulevenshtein
```
Comment (mrakitin) [inline recipes-tag/xpdan/meta.yaml]: @CJ-Wright, why was this reverted? I think we decided to remove it in the earlier PRs (#729).

Comment (CJ-Wright) [reply recipes-tag/xpdan/meta.yaml]: Sorry I completely missed that

Comment (mrakitin) [reply recipes-tag/xpdan/meta.yaml]: No worries, we tracked this down and fixed via #737. The `xpd*` packages are compatible with the `*-2019-3.0` envs.

Comment (CJ-Wright) [reply recipes-tag/xpdan/meta.yaml]: Thank you so much!

**Pull request merged**